src/Components/DetailComponents/HotelCalendar.js

```diff
@@ -0,0 +1,379 @@
+            }}
+            onClick={(e) => e.stopPropagation()}
+          >
+            <DatePicker
+              className="endPicker"
+              locale="ko"
+              selected={startDate}
+              onChange={(date) => setEndDate(date)}
+              minDate={startDate}
+              inline
+            />
+          </div>
+        </div>
+      </div>
+      <div
+        className="numberPeopleBtn"
+        onClick={() =>
+          modalChoice !== "update"
+            ? setModalChoice("update")
+            : setModalChoice("allClose")
+        }
+      >
+        <CheckIn>
+          <i className="fas fa-users" />
+          <div className="checkColumn">
+            <p className="checkTitle">인원수</p>
+            <p className="checkDate">
+              객실 {roomInput}개,성인 {peopleInput}명
+            </p>
+          </div>
+        </CheckIn>
+        <div
+          className="personModal"
+          style={{ visibility: `${modalChoice === "update" ? "" : "hidden"}` }}
+          onClick={(e) => e.stopPropagation()}
+        >
+          <div className="modalImgContainer">
+            <i
+              className="fas fa-times fa-2x"
+              onClick={() =>
+                modalChoice !== "update"
+                  ? setModalChoice("update")
+                  : setModalChoice("allClose")
+              }
+            />
+          </div>
+          <div className="inputContainer">
+            <div className="modalTitle">
+              <i className="fas fa-bed" />
+              객실
+            </div>
+            <div className="caculatar">
+              <button
+                className="numberBtn"
+                name="minus"
+                id="roomInput"
+                onClick={(e) => reservationHandler(e)}
+              >
+                -
+              </button>
+              <input className="productNumber" value={roomInput} readOnly />
+              <button
+                className="numberBtn"
+                name="plus"
+                id="roomInput"
+                onClick={(e) => reservationHandler(e)}
+              >
+                +
+              </button>
+            </div>
+          </div>
+          <div className="inputContainer">
+            <div className="modalTitle">
+              <i className="fas fa-user-friends" />
+              성인
+            </div>
+            <div className="caculatar">
+              <button
+                className="numberBtn"
+                name="minus"
+                id="peopleInput"
+                onClick={(e, peopleInput) => reservationHandler(e, peopleInput)}
+              >
+                -
+              </button>
+              <input className="productNumber" value={peopleInput} readOnly />
+              <button
+                className="numberBtn"
+                name="plus"
+                id="peopleInput"
+                onClick={(peopleInput, e) => reservationHandler(peopleInput, e)}
```

Comment:
```suggestion
                onClick={reservationHandler}
```
인자로 넘겨줄 필요 없을 것 같습니다

Reply:
수정완료